### PR TITLE
GitHub action for updating plugin stats

### PR DESCRIPTION
**Pull request by ericaxu**
@liamcain @argenos If you have time to give some advice!

Goal of the workflow is to update plugin stats and commit back to the repo every day with a nodejs script.

### CODE REVIEW

File: .github/workflows/plugin-stat.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - cron: "0 0 * * *"
+
+jobs:
+  pull-stats:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
```
Comment (liamcain) [inline .github/workflows/plugin-stat.yml:14]: Maybe use v16? 14 is pretty old 

```suggestion
          node-version: 16.x
```

File: .github/workflows/plugin-stat.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          node update-stats.js
+  commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Commit updates
+        run: |
+          git config --local user.name 'Obsidian Bot'
+          git config --local user.email 'admin@obsidian.md'
+          git commit -am "Update plugin stats"
```
Comment (liamcain) [inline .github/workflows/plugin-stat.yml:27]:
```suggestion
          git commit -am "chore: Update plugin stats"
```

Just a suggestion to make it a semantic commit message.

**Review by liamcain — APPROVED** [review]
Looks good to me! Just left a couple of suggestions but I'm excited to see the stats change every day!

**Comment by ericaxu** [comment]
> Looks good to me! Just left a couple of suggestions but I'm excited to see the stats change every day!

Thanks! Finger crossed it will work tonight.

**Comment by ericaxu** [comment]
Sadly it had a bunch of errors. Will resolve them and test again.

**Pull request merged**